App.js

```diff
@@ -30,10 +32,74 @@ import {
 import { Colors } from "react-native/Libraries/NewAppScreen";
 
 function HomeScreen() {
+  const [length, setLength] = useState(0);
+  const [width, setWidth] = useState(0);
+  const [gram, setGram] = useState(0);
+  const [currentCount, setCurrentCount] = useState(0);
```

Comment:
name "currentCount" doesn't explain what value this state is holding to someone new.
```suggestion
  const [currentSheetsCount, setCurrentSheetsCount] = useState(0);
```